knotx-core/knotx-engine-view/src/main/java/com/cognifide/knotx/engine/view/parser/TemplateHtmlFragment.java

```diff
@@ -45,21 +42,17 @@
 public class TemplateHtmlFragment implements HtmlFragment {
 
   private static final String DATA_SERVICE = "data-service.*";
-
   private static final String DATA_PARAMS = "data-params.*";
 
-  private static final String DATA_ID = "data-id";
-
-  private String content;
-
-  private String dataId = StringUtils.EMPTY;
+  private final String content;
```

Comment:
so many content words here - maybe call it more meaningful, e.g. unwrappedFragmentContent (just to say it's actually fragment content, but unwrapped - without script tags..